autoload/firenvim.vim

```diff
@@ -273,16 +272,19 @@ function! firenvim#install(...)
                 echo "Installed native manifest for " . l:name . "."
 
                 if has('win32')
-                        echo "Creating registry key for " . l:name . ". This may take a while."
                         " On windows, also create a registry key. We
                         " do this by writing a powershell script to a
                         " file and executing it.
                         let l:ps1_content = s:key_to_ps1_str(l:cur_browser["registry_key"],
                                                 \ l:manifest_path)
                         let l:ps1_path = s:build_path([l:manifest_dir_path, l:name . ".ps1"])
+                        echo "Creating registry key for " . l:name . ". This may take a while. Script: " . l:ps1_path
                         call writefile(split(l:ps1_content, "\n"), l:ps1_path)
                         call setfperm(l:ps1_path, "rwx------")
-                        call system('powershell "' . l:ps1_path . '"')
+                        let o = system(['powershell', '-Command', '-'], readfile(l:ps1_path))
```

Comment:
Even though the `.ps1` script is not strictly necessary after this change, I left it since it is useful as a reference.  Though I would prefer in general to capture things in a log rather than have a bunch of files to inspect.

Reply:
You're right, I should probably implement proper logs, both in the browser plugin and in the neovim plugin.

Does neovim have logging facilities (besides :echo/:echoerr) or do plugin developers need to build their own?

Reply:
> Does neovim have logging facilities (besides :echo/:echoerr) or do plugin developers need to build their own?

It's still a (much needed) WIP/todo. It's one of my next priorities.